style.css

```diff
@@ -0,0 +1,45 @@
+body,
+ul {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-image: url('./img/bg_sinamo.jpg');
```

Comment:
지금 실행결과 보면 배경사진이 반복되어 있고,  배경사진이 잘려있는데 이것을 코드로 반복하지 않게 설정할 수 있어요

자세한 건
[css background mdn](https://developer.mozilla.org/ko/docs/Web/CSS/background) 이곳을 보거나,
아래 코드를 구글링해서 살펴보도록 해요!
```suggestion
    background-image: url('./img/bg_sinamo.jpg'); 
    background-position: center; // 배경 사진 가운데
    background-repeat: no-repeat; // 반복하지 않도록
```
![image](https://user-images.githubusercontent.com/87893624/149653802-c9258b8f-d5eb-4148-8e4d-6c0b64ab32c5.png)